pages/api/webhooks/order-created.ts

```diff
@@ -1,23 +1,55 @@
+import { SALEOR_DOMAIN_HEADER } from "@saleor/app-sdk/const";
 import {
   withRegisteredSaleorDomainHeader,
   withSaleorEventMatch,
   withWebhookSignatureVerified,
 } from "@saleor/app-sdk/middleware";
 import { withSentry } from "@sentry/nextjs";
-import type { Handler } from "retes";
+import { Handler } from "retes";
 import { toNextHandler } from "retes/adapter";
 import { Response } from "retes/response";
 
+import { getValue } from "../../../lib/metadata";
 import { apl } from "../../../lib/saleorApp";
 
 const handler: Handler = async (request) => {
-  //
-  // Your logic goes here
-  //
-  console.debug("Received order webhook.");
-  console.debug(request.body);
+  const saleorDomain = request.headers[SALEOR_DOMAIN_HEADER];
+  const webhookUrl = await getValue(saleorDomain as string, "WEBHOOK_URL");
 
-  return Response.OK({ success: true });
+  const context = request.params;
+  const {
+    order: { id, userEmail },
+  } = context;
+
+  if (!id) {
+    return Response.BadRequest({ success: false, message: "No order id." });
+  }
+
+  const response = await fetch(webhookUrl, {
```

Comment:
Could be moved to `/backend/slack.ts` as for example `sendSlackMessage`.

Will be easier to reuse logic and IMO keeping business logic out of the handler improve the maintenance of the app.

Reply:
Sure - I was thinking if I should move to its own package but I did not do it as for the time being I don't need to call Slack API in other places. 

Anyway I will move to to its own module - what about using existing `lib` folder? I will be a bit up to date with saleor-app-klaviyo [structure](https://github.com/saleor/saleor-app-klaviyo/tree/main/lib).